src/screens/SignTransactionSheet.tsx

```diff
@@ -2126,6 +2131,11 @@ const SimulatedEventRow = ({
           <Text color="label" size="17pt" weight="bold">
             {eventInfo.label}
           </Text>
+          {showUSD && (
+            <Text color="labelTertiary" size="15pt" weight="bold">
+              {formattedPrice}
+            </Text>
+          )}
```

Comment:
```suggestion
          <Inline alignVertical="bottom" space="6px">
            <Text color="label" size="17pt" weight="bold">
              {eventInfo.label}
            </Text>
            {showUSD && (
              <Text color="labelQuaternary" size="12pt" weight="bold">
                {formattedPrice}
              </Text>
            )}
          </Inline>
```

Reply:
![image](https://github.com/rainbow-me/rainbow/assets/6868432/40277ffd-7374-4c1e-9143-bb42c8072cec)
12 seems small but if you're good with it, i committed it

Reply:
you can bump the text size up to 13pt

re: your latest changes, the additional `Inline` should wrap both the `eventInfo.label` and `formattedPrice`, like this:

```
<Inline alignVertical="bottom" space="6px" wrap={false}>
  <Text color="label" size="17pt" weight="bold">
    {eventInfo.label}
  </Text>
  {showUSD && (
    <Text color="labelQuaternary" size="13pt" weight="bold">
      {formattedPrice}
    </Text>
  )}
</Inline>
```

doesn't have any effect if it only wraps a single child

Reply:
that is good to know, ty - fixed